webpack.config.js

```diff
@@ -0,0 +1,99 @@
+const path = require('path');
+const webpack = require('webpack');
+const MiniCssExtractPlugin = require('mini-css-extract-plugin');
+const CssMinimizerPlugin = require('css-minimizer-webpack-plugin');
+const { CleanWebpackPlugin } = require('clean-webpack-plugin');
+
+module.exports = (env, argv) => {
+  const { mode } = argv;
+
+  return {
+    devtool: mode === 'production'
+      ? 'source-map'
+      : 'cheap-module-eval-source-map',
+    entry: {
+      'fieldmanager': [
+        './js/fieldmanager.js',
+        './css/fieldmanager.css',
+        './css/fieldmanager-options.css',
+      ],
+      'fieldmanager-autocomplete': './js/fieldmanager-autocomplete.js',
+      'fieldmanager-colorpicker': './js/fieldmanager-colorpicker.js',
+      'fieldmanager-datepicker': './js/fieldmanager-datepicker.js',
+      'fieldmanager-draggablepost': [
+        './js/fieldmanager-draggablepost.js',
+        './css/fieldmanager-draggablepost.css',
+      ],
+      'fieldmanager-group-tabs': [
+        './js/fieldmanager-group-tabs.js',
+        './css/fieldmanager-group-tabs.css',
+      ],
+      'fieldmanager-media': './js/media/fieldmanager-media.js',
+      'fieldmanager-select': './js/fieldmanager-select.js',
+      'fieldmanager-quickedit': './js/fieldmanager-quickedit.js',
+      'fieldmanager-options': './css/fieldmanager-options.css',
+      'richtext': './js/richtext.js',
+    },
+    output: {
+      filename: mode === 'production'
+        ? 'js/[name].bundle.min.js'
+        : 'js/[name].js',
+      path: path.join(__dirname, 'build'),
+    },
+    module: {
+      rules: [
+        {
+          test: /.css$/,
+          use: [
+            MiniCssExtractPlugin.loader,
+            'css-loader',
+            {
+              loader: 'postcss-loader',
+              options: {
+                postcssOptions: {
+                  plugins: [
+                    [
+                      // postcss-preset-env includes autoprefixer
+                      'postcss-preset-env',
+                      {
+                        browsers: 'last 2 versions',
```

Comment:
This may need to be adjusted but for the most cases the -webkit and -moz prefixes are added here with autoprefixer

Reply:
Consider adding this to `package.json` via the `browserslist` field (or a `.browserslistrc` file) instead so other tools like Babel can make use of it too. 🙂 

https://github.com/csstools/postcss-preset-env#browsers
https://babeljs.io/docs/en/babel-preset-env